v2/lpwm-csp/index.html

```diff
@@ -0,0 +1,360 @@
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="user-scalable=0, width=device-width, initial-scale=1.0,  minimum-scale=1.0, maximum-scale=1.0">
+    <meta http-equiv="Content-Security-Policy" content="connect-src self wss://*.msg.liveperson.net">
```

Comment:
This is the actual change - strict CSP policy to block fetch requests.
Everything else is just a copy of existing page